ramls/ramls.raml

```diff
@@ -0,0 +1,38 @@
+#%RAML 1.0
+
+title: RAML API
+version: v1
+baseUri: http://localhost:8081/{version}
+
+documentation:
+  - title: RAML API
+    content: This documents the API calls that can be made to get RAML of a module defined by X-Okapi-Module-Id header
+
+types:
+  errors: !include ../schemas/errors.schema
+
+/_/ramls:
+  get:
+    description: Get RAMLs for the given module defined by header X-Okapi-Module-Id
```

Comment:
Explain the two cases in the description: List all raml paths, return a raml file at a specific path.